Mods/ModVarietyPack/Defs/ThingDefs/Plants_Cultivated.xml

```diff
@@ -61,8 +61,8 @@
       <soundHarvesting>Harvest_Standard</soundHarvesting>
       <soundHarvestFinish>Harvest_Standard_Finish</soundHarvestFinish>
       <sowWork>150</sowWork>
-      <!--<fertilityMin>0.5</fertilityMin>
-      <fertilityFactorGrowthRate>1.0</fertilityFactorGrowthRate>-->
+      <fertilityMin>0.5</fertilityMin>
+      <fertilityFactorGrowthRate>1.0</fertilityFactorGrowthRate>
```

Comment:
Actually this was not to come through... Was testing to see if the native grass would grow. it still does not. Playing in a temperate forest, in 30 game days looks like I am playing in the arid shrublands, all the native animals are starving, which means my colonist will also be starving soon, with nothing left to hunt. 

Don't know what you changed to cause this problem, but it really needs to be found and fixed.